account_invoice_constraint_chronology/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Account Invoice Constraint Chronology
+=====================================
+
+This module helps ensuring the chronology of invoice numbers.
+
+It prevents the validation of invoices when:
+
+* there are draft invoices with an anterior date
+* there are validated invoices with a posterior date
+
+Configuration
+=============
+
+To configure this module, go to the menu *Accounting > Configuration > Journals > Journals* and activate the option *Check Chronology* on the relevant journals. After the installation of the module, this option will be active on *sale* and *sale refund* journals.
```

Comment:
Replace *Accounting > Configuration > Journals > Journals*
for *Accounting/Invoicing > Configuration > Accounting > Journals*